src/Symfony/Bundle/FrameworkBundle/Controller/TemplateController.php

```diff
@@ -33,8 +33,14 @@ class TemplateController extends ContainerAware
      */
     public function templateAction($template, $maxAge = null, $sharedAge = null, $private = null)
     {
-        /** @var $response \Symfony\Component\HttpFoundation\Response */
-        $response = $this->container->get('templating')->renderResponse($template);
+        /* @var $response \Symfony\Component\HttpFoundation\Response */
```

Comment:
It was to make fabbot green, I've tried with PHPStorm, both formats enable the autocompletion. Removing it prevent autocompletion by default, but if you have the Symfony plugin then it works.